core/connection/interface.go

```diff
@@ -57,7 +58,7 @@ type PromiseIssuerCreator func(issuerID identity.Identity, dialog communication.
 // Manager interface provides methods to manage connection
 type Manager interface {
 	// Connect creates new connection from given consumer to provider, reports error if connection already exists
-	Connect(consumerID, accountantID identity.Identity, proposal market.ServiceProposal, params ConnectParams) error
+	Connect(consumerID identity.Identity, accountantAddress common.Address, proposal market.ServiceProposal, params ConnectParams) error
```

Comment:
This is very confusing to name it `accountantAddress` because we have CLI argument called `accountant.address` which means `accountant URL address`:
https://github.com/mysteriumnetwork/node/blob/master/config/flags_accountant.go#L27-L31

So the accountant address is an accountant URL address. Can we keep calling it `accountantID`?

Reply:
I agree that address is a broad term

Reply:
This scope is not related to configuring stage there `accountant.address` is used.

But I reverted this name change `accoutantAddress -> accountantID` , as this accountant unique identifier is already exposed to public contracts as `accountant_id`